src/lib/functions/runtimes/js/worker.mjs

```diff
@@ -0,0 +1,52 @@
+import { createServer } from 'net'
+import { isMainThread, workerData, parentPort } from 'worker_threads'
+
+import lambdaLocal from '@skn0tt/lambda-local'
+import { isStream } from 'is-stream'
+import sourceMapSupport from 'source-map-support'
+
+import { SECONDS_TO_MILLISECONDS } from './constants.mjs'
+
+if (isMainThread) {
+  throw new Error(`Do not import "${import.meta.url}" in the main thread.`)
+}
+
+sourceMapSupport.install()
+
+lambdaLocal.getLogger().level = 'warn'
+
+const { clientContext, event, lambdaPath, timeout } = workerData
+
+// eslint-disable-next-line import/no-dynamic-require
```

Comment:
I think we can disable this globally.

Reply:
done in https://github.com/netlify/cli/pull/5804/commits/8273e8a0553aa23b5003db6abe3228848ebe3354